src/signature.zig

```diff
@@ -0,0 +1,276 @@
+/// Number of random bytes used for verification.
+const RAND_BYTES = 8;
+
+/// Number of random bits used for verification.
+const RAND_BITS = 8 * RAND_BYTES;
+
+/// Verify multiple aggregate signatures efficiently using random coefficients.
+///
+/// Source: https://ethresear.ch/t/fast-verification-of-multiple-bls-signatures/5407
+///
+/// Returns true if verification succeeds, false if verification fails, `BlstError` on error.
+pub fn verifyMultipleAggregateSignatures(
+    pairing_buf: *[pairing_size]u8,
+    n_elems: usize,
+    msgs: [*c]const [32]u8,
+    dst: []const u8,
+    pks: [*c]const *PublicKey,
+    pks_validate: bool,
+    sigs: [*c]const *Signature,
+    sigs_groupcheck: bool,
+    rands: [*c]const [32]u8,
```

Comment:
`rands` here is a different type than `randomness` in AggregateSignature, is this intentional?